.dockstore.yml

```diff
@@ -129,3 +129,8 @@ workflows:
    primaryDescriptorPath: /PECGS-QUICviz/QUICviz.wdl
    testParameterFiles:
     - /PECGS-QUICviz/QUICviz.inputs.json
+ - name: CoverageProfiler
```

Comment:
Can you add a CoverageProfiler.md describing what the pipeline does?

Reply:
I will add a README.md page.